dll/win32/msgina/branding.c

```diff
@@ -58,4 +63,261 @@ Brand_Cleanup(
     ZeroMemory(pBrand, sizeof(*pBrand));
 }
 
+// HACK, now used only to resize the overall window.
+static VOID
+AdjustWindowSize(
+    _In_ HWND hWnd,
+    _In_ PBRAND pBrand,
+    _In_ INT xDelta,
+    _In_ INT yDelta)
+{
+    INT xOld, yOld, cxOld, cyOld;
+    INT xNew, yNew, cxNew, cyNew;
+    RECT rc;
+    // DWORD style, exstyle;
+
+    /* Get "current" (-> old) window position and size */
```

Comment:
Hmm I wonder what would be a non-hack way to move all controls.

Reply:
"HACK" was refering to the previous purpose of the function (that was introduced by Katahiro to fixup the design of the dialog in japanese language only. Then I repurposed the function but it still needed cleanup, and I kept my own comment that it was a hack. I think I've now sanitized it, if you look at the code as a whole now (not at some intermediate commit).